app/scaffold-controllers/cart/cartModalController.js

```diff
@@ -81,11 +81,13 @@ function(
         this.isShowing = true;
         this._reload();
         this.viewPlugin.show({animated: true});
+        Astro.events.trigger('cart:shown');
```

Comment:
Create an app event bus for eventing throughout the app.js file. (Just adding it for visibility...but we already discussed this in person.)